rumqttd/Cargo.toml

```diff
@@ -20,10 +20,10 @@ flume = { version = "0.11.0", default-features = false, features = ["async"]}
 slab = "0.4.9"
 thiserror = "1.0.49"
 tokio-util = { version = "0.7", features = ["codec"], optional = true }
-tokio-rustls = { version = "0.24", optional = true }
-rustls-webpki = { version = "0.101.6", optional = true }
-tokio-native-tls = { version = "0.3", optional = true }
-rustls-pemfile = { version = "1", optional = true }
+tokio-rustls = { version = "0.25.0", optional = true }
+rustls-webpki = { version = "0.102.1", optional = true }
+tokio-native-tls = { version = "0.3.1", optional = true }
```

Comment:
would it be better to not include the PATCH version. basically only like `0.25` and `0.3` or `2`?

Reply:
Why? I only tested with this version so there is no guarantee that an earlier version would work, and specifying the full versions still allows newer semver-compatible versions. E.g. `0.3.1` allows 0.3.1 <= version < 0.4.0, whereas `0.3` allows 0.3.0 <= version < 0.4.0.

Reply:
gotcha, thanks for clarifying 😃